### PR TITLE
Create student-program.md

### PR DESCRIPTION
**Pull request by dphow**
I still need to add links for registering to pre-workshop session. 

Also, abstract for that session may be "too long" to include outright on the page so let me know if you suggest as well paring it down. 

Lastly, I was having issues installing jekyll on my machine to test how it looks. Will add a comment here once I produce a test page to confirm how it looks.

## Checklist:

- [ ] (Committee Chairs Only) I have posted the link for the PR in the usrse slack (#usrse-2024-conference-planning-committee) to ask for content reviewers
- [x] I have previewed changes locally
- [x] I have updated the README.md if necessary

cc **ADD GITHUB HANDLES HERE**

### CODE REVIEW

File: pages/program/abstracts/student-program.md [hunk 1]
```diff
@@ -0,0 +1,50 @@
+---
```
Comment (mrmundt) [inline pages/program/abstracts/student-program.md:1]: There will need to be more changes in this PR to actually make it show up in the page tree:
- https://github.com/USRSE/usrse24/blob/main/_data/navigation.yml
- https://github.com/USRSE/usrse24/blob/main/_data/menus/program.yml

**Review by mrmundt — CHANGES_REQUESTED** [review]

File: pages/program/abstracts/student-program.md (hunk 1 above)
Comment (dphow) [reply pages/program/abstracts/student-program.md:1]: Done in two most recent commits

File: pages/program/abstracts/student-program.md [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+
+Individuals who participate in this program will be invited to submit screenshots 
+of their digital presence portfolio to be featured during the US-RSE’24 conference 
+poster session.
+
+## Student Ice Cream Social
+Tuesday, October 15th 6:00 to 8:00 PM 
+ 
+Wrap up the first conference day experience by joining us at 
+end of day poster session. This is a great opportunity to meet
```
Comment (mrmundt) [inline pages/program/abstracts/student-program.md:35]:
```suggestion
the end of the poster session. This is a great opportunity to meet 
```

Comment (mrmundt) [reply pages/program/abstracts/student-program.md:35]: This is my only final suggestion because the phrasing is just a wee bit weird.

**Comment by dphow** [comment]
@mrmundt In case you have notifications turned off from PR threads, a gentle reminder here.

Also, let me know if you'd like me to add directly via a separate PR the details for Managing Digital Presence workshop in the `program.md` and `workshops`md` files. Not sure how best to edit the program page though as that is saved in repo as HTML and I don't know if it got generated from other repo files by jekyll, etc.

**Comment by dphow** [comment]
My bad, just saw your other comment :)

**Pull request merged**